.github/issue_template.md

```diff
@@ -0,0 +1,36 @@
+<!---
+ Please use this template to fill your issue, issues that does not
+ follow this will be asked to be edited to match the template, or will be closed
```

Comment:
```suggestion
 Please use this template to fill your issue. Issues that does not
 follow this will be asked to be edited to match the template, or will be closed.
```

Reply:
Fill in*